roles/filebeat/templates/filebeat.yml.j2

```diff
@@ -0,0 +1,14 @@
+filebeat.prospectors:
+- input_type: log
+  paths:
+    - /mnt/log/*.log
+fields_under_root: true
+fields:
+  hostname: {{ inventory_hostname }}
+output.kafka:
```

Comment:
in order to make this useful for general mantl installations, I'm thinking that the kafka output should be optional and configurable. We'd also want people to be able to use this with elasticsearch, for example.